src/capture/CMakeLists.txt

```diff
@@ -11,11 +11,18 @@ add_library(libcapture STATIC
 		image/png_writer.cpp
 )
 
-find_package(PNG CONFIG REQUIRED)
+if (PKG_CONFIG_FOUND)
+	pkg_check_modules(PNG REQUIRED IMPORTED_TARGET libpng)
+	target_include_directories(libcapture PRIVATE ${PNG_INCLUDE_DIRS})
+	target_compile_options(libcapture PRIVATE ${PNG_CFLAGS_OTHER})
+	target_link_libraries(libcapture PRIVATE ${PNG_LIBRARIES})
+else()
+	find_package(PNG CONFIG REQUIRED)
+	target_link_libraries(libcapture PRIVATE PNG::PNG)
+endif()
```

Comment:
This shouldn't be needed for the majority of our dependencies.  CMake ships with a bunch of Find*.cmake files to make `find_package()` work for system deps.  In Debian, the libpng one is here for example `/usr/share/cmake-3.25/Modules/FindPNG.cmake`

I had to change this line from `find_package(PNG CONFIG REQUIRED)` to `find_package(PNG REQUIRED)`.  More info on the `CONFIG` parameter here:  https://cmake.org/cmake/help/latest/command/find_package.html#full-signature

@kklobe must have added `CONFIG` to his PR after I tested an early version of it.  If I'm reading the docs right, I think it effectively disables searching system libs so we may want to (conditionally?) remove that flag.

Reply:
I could use _CMake_ to find _libpng_, _OpenGL_, _SDL2_net_, and _ALSA_. Unfortunately, the flag has to stay - _iir_, _OpusFile_, _MT32Emu_, and _FluidSynth_ can't be found this way.

Reply:
I added `CONFIG` because that forces `find_package` to use the *Config.cmake or -config.cmake file that comes with the vcpkg dep instead of the generic one included with CMake, or finding the system-installed version, so I figured any package that we are installing with vcpkg should be configured with `CONFIG` so we are sure to pull it from vcpkg specifically.